packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -157,10 +157,32 @@ static inline BOOL IsHorizontalNavigationWithinZoneAction(FocusZoneAction action
 	return nil;
 }
 
+/// Bypass FocusZone if it's empty or has no focusable elements
+static BOOL ShouldSkipFocusZone(NSView *view)
+{
+    if([view isKindOfClass:[RCTFocusZone class]])
+    {
+        // FocusZone has no elements
+        if ([[view subviews] count] < 1)
+        {
+            return YES;
+        }
+        
+        NSView* keyView = GetFirstKeyViewWithin(view);
+        // FocusZone has no focusable elements
+        if (keyView == nil)
+        {
+            return YES;
+        }
+    }
+
+    return NO;
+}
+
 - (BOOL)acceptsFirstResponder
 {
 	// Accept firstResponder on FocusZone itself in order to reassign it within the FocusZone.
```

Comment:
it seems like the comment should be updated?

Reply:
I think this comment states the justification for overriding `acceptsFirstResponder`, though I think putting it above the function header would make more sense.

Reply:
The comment was added when Chris Pearson added support for `defaultTabbableElement` (before I think we just didn't override it). I also think it's worth updating the comment to clarify what cases we do/don't accept first responder.